catalog/core/catalog-core-validationfilterplugin/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -19,9 +19,9 @@
     <OCD name="Validation PreFederated Filter Plugin"
          id="org.codice.ddf.catalog.plugin.validationfilter.ValidationFilterPlugin">
         <AD
-                description="Mapping of SECURITY attribute to user attribute."
+                description="Comma delimited mapping of SECURITY attribute to user attribute."
```

Comment:
✏️ Should be "comma-delimited".

Reply:
I kinda wanna add a `NOTE: commas must be escaped.`

Reply:
The commas don't have to be escaped after saving with new values, only in the MetaType default value.